packages/ozone/src/db/schema/ozone_set.ts

```diff
@@ -0,0 +1,24 @@
+import { Generated, GeneratedAlways } from 'kysely'
+
+export const ozoneSetTableName = 'set_detail'
+export const ozoneSetValueTableName = 'set_value'
+
+export interface SetDetail {
+  id: GeneratedAlways<number>
+  name: string
+  description: string | null
+  createdAt: Generated<Date>
+  updatedAt: Generated<Date>
+}
+
+export interface SetValue {
+  id: GeneratedAlways<number>
+  setId: number
+  value: string
+  createdAt: string
```

Comment:
It looks like this column is also `Generated` based on the migration.  However, it looks like in practice we mostly aren't relying on postgres to set these default values—I think it's only used for `SetDetail.createdAt`.  One option is to leave those defaults in postgres as a backstop, but remove them from here in the kysely model and always set these values explicitly.

Reply:
yeah i was seeing some type errors and didn't wanna f around so just moved it to string but Generated<Date> also works and went with that.

Reply:
to clarify, still inserting the date values from JS and not really depending on postgres.